docs/sources/administration/feature-management/index.md

```diff
@@ -0,0 +1,44 @@
+---
+description: Describes how admin users can manage feature toggles
+keywords:
+  - feature
+  - toggle
+labels:
+  products:
+    - enterprise
+    - oss
+title: Feature Toggles Management
+weight: 900
+menuTitle: Feature Management
+---
+
+# Feature Toggles
+
+Grafana incorporates feature toggles, enabling the introduction of new functionalities safeguarded by a control flag. This allows Grafana administrators the flexibility to determine the appropriate timing for enabling or disabling specific features.
+For detailed insights into particular features and how they operate, please consult [Configure Feature Toggles](/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/").
+
+## Manage Feature Toggles
+
+In the Administration page, the Feature Management section enables administrators to see and switch the various features active in their Grafana environment. Based on the development stage of the feature, we establish specific options for visibility and customization.
+
+| Stage                | Visibility | Editable |
+| -------------------- | ---------- | -------- |
+| Experimental         | Hidden     | No       |
+| PrivatePreview       | Visible\*  | No       |
+| Public Preview       | Visible\*  | No       |
+| General Availability | Visible\*  | Yes\*    |
+
+**Note:** Options marked with an asterisk (\*) are defaults for the corresponding feature stage. Each feature toggle owner can override its default behaviour.
```

Comment:
There is a shortcode for consistent admonitions: https://grafana.com/docs/writers-toolkit/write/shortcodes/#admonition

Prefer U.S. English (behavior, not behaviour). 

```suggestion
{{< admonition type="note" >}}
Options marked with an asterisk (\*) are defaults for the corresponding feature stage.
Each feature toggle owner can override its default behavior.
{{< /admonition >}}
```